docs/add-ons/external-dns.md

```diff
@@ -22,9 +22,10 @@ Alternatively, you can supply a list of Route53 zone ARNs which external-dns wil
 
 ```hcl
   external_dns_route53_zone_arns = [
-    "arn:aws:route53::123456789012:hostedzone/Z1234567890"
+    "arn:aws:route53:::hostedzone/Z1234567890"
```

Comment:
if you remove this we don't know where to put the account id

Reply:
From the markdown ?

Reply:
@allamand from terraform

Reply:
from AWS API response